mvcc/backend/batch_tx.go

```diff
@@ -159,6 +161,12 @@ func (t *batchTx) commit(stop bool) {
 	var err error
 	// commit the last tx
 	if t.tx != nil {
+		if t.stopped {
```

Comment:
dropping a commit on the floor like this without an error and acting like it worked is kind of scary; what if the commit is important?